crm/optimize/gamification.rst

```diff
@@ -0,0 +1,86 @@
+==========================================
+How to motivate and reward my salespeople?
+==========================================
+
+Challenging your employees to reach specific targets with goals and rewards is an excellent way to
+reinforce good habits and improve your salespeople productivity. The Gamification module gives you
+simple and creative ways to motivate and evaluate your employees with real-time recognition and
+badges inspired by game mechanics.
+
+Configuration
+=============
+
+Install the *Gamification* module, or the *CRM gamification* one, which adds some useful data
+(goals and challenges) that can be used on *CRM/Sale*.
+
+.. image:: media/gamification_module_install.png
+   :align: center
+   :alt: View of the gamification module being installed in Odoo
+
+Create a challenge
+==================
+
+Create now your first challenge by going to menu :menuselection:`Settings --> Gamification Tools
+--> Challenges`.
+
+.. note::
+   As the gamification tool is a one-time technical setup, you will need to activate the technical
+   features in order to access the configuration. In order to do so, go to the *Settings* and
+   *Activate the developer mode*.
+
+.. image:: media/gamification_tools_menu.png
+   :align: center
+   :alt: View if the gamification tools menu in Odoo Settings
+
+A challenge is a mission that you send to your sales team. It can include one or several goals, and
+it is set for a specific period of time. Configure your challenge as follows:
```

Comment:
```suggestion
A challenge is a mission that you send to your sales team. It can include one or several goals, 
set for a specific period of time. Configure your challenge as follows:
```